dashboards-notebooks/public/components/notebook.tsx

```diff
@@ -935,7 +952,7 @@ export class Notebook extends Component<NotebookProps, NotebookState> {
             {this.state.parsedPara.length > 0 ? (
               <>
                 <Cells>
-                  <PanelWrapper shouldWrap={this.state.selectedViewId === 'output_only'}>
```

Comment:
this removes panel completely in output only view?

Reply:
I'm removing the `shouldWrap` parameter- the old implementation is what causes the stray `EuiPanel` in the output only view.